addons/web/static/lib/qweb/tests.js

```diff
@@ -0,0 +1,77 @@
+odoo.define('qweb.tests', function () {
+'use strict';
+
+function trim(s) {
```

Comment:
what is the difference between this and String.prototype.trim? (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim)

Reply:
I would expect none, this was just copied from the original file unmodified.

Reply:
PS: I sent you a mail yesterday, not sure you've seen it.